internal/clientconn/conn_metrics.go

```diff
@@ -70,6 +93,83 @@ func (cm *ConnMetrics) Collect(ch chan<- prometheus.Metric) {
 	cm.aggregationStages.Collect(ch)
 }
 
+func (cm *ConnMetrics) Responses() map[string]CommandMetrics {
+	res := make(map[string]CommandMetrics)
+
+	// initialize commands in the map to show zero values in the metrics output
+	for k := range common.Commands {
```

Comment:
Here (`clientconn` package) we need to import `common.Commands` to fill the map with all of the listed commands